src/search/components/query-editor-modal.html

```diff
@@ -14,17 +20,17 @@ <h2 class="modal-title">Geavanceerde zoekopdracht bouwer</h2>
       <div class="panel-heading">
         <div class="row">
           <div class="col-sm-3">
-            <h3 class="panel-title">Groep <span ng-bind="$index + 1"></span></h3>
+            <h3 class="panel-title"><span translate-once="search.advancedQueryBuilder.group">Groep </span><span ng-bind="$index + 1"></span></h3>
```

Comment:
The text between the tags can be removed I guess?

Reply:
I left the text to give more context so that when you have to change something it is still clear what it is about, but it gets overwritten when it is translated.

I don't know what is preferred.

Reply:
I would prefer without.
@bertramakers what do you think?

Reply:
I will remove them

Reply:
I can see a benefit in both ways, but I think without is also slightly better because the text can get out of date and/or it can get confusing, for example someone might think that the text inside the tag is the Dutch and the translate only happens for other languages. But in any case this app has a short life ahead of it so it doesn't matter that much in the scheme of things.